README.md

```diff
@@ -1,2 +1,50 @@
-# README
-The README for the makingitallwork repos
+# Makingitallwork Introduction
+
+The makingitallwork repositories are a hobby project to provide an example platform for educational purposes.  This platform will test, build and deploy the sample applications to infrastructure that is managed via infrastructure  as code.  
+
+## 10,000 Foot View
+At a very high level this project has 4 main pieces
+
+1.  Base Container
+2.  Applications
+3.  Infrastructure Code
+4.  Infrastructure
+
+Here is a diagram of how all these pieces fit together.  Please not the number of applications is just an example and holds no real significance.
+
+```mermaid
+
+graph TD;
+    Base_Image --> App_A;
+    Base_Image --> App_B;
+    Base_Image --> App_C;
+    App_A --> Infrastructure_Code;
+    App_B --> Infrastructure_Code;
+    App_C -->  Infrastructure_Code;
+    Infrastructure_Code -->  Infrastructure;
+```
+
+### 1. Base Container 
+
+_Containers are just a CVE delivery system  
+-- Unknown_
+
+A base container image (a.k.a golden image) is used to create a standard image for an organization that will contain tools that are standard (expected to be there) throughout an organization. 
+
+Why would you use a base image?  Ideally it allows you to move change at an organizational level.  There are several scenarios where a base image is advantageous.
```

Comment:
In reading through this I think this breaks the flow.  We should try to present information going from least specific to most specific.  This explains a number of details which feels like a tangent from the purpose.  It would be better served to have specific information like this in the base container image instead.  This way people can get just the high level here and if they have interest they can go to the base container repo to learn more.